npm start fails

When I was trying to develop for this project, I ran the build instructions as included on the `README`, however I got the following error:

```sh
-> % npm run build

> twock@0.1.0 build
> next build

info  - Linting and checking validity of types  
info  - Creating an optimized production build  
Failed to compile.

./pages/[stock].js
Module not found: Can't resolve 'pg' in '/Users/ben/Downloads/test/twock/pages'

./pages/index.js
Module not found: Can't resolve 'stock-ticker-symbol' in '/Users/ben/Downloads/test/twock/pages'


> Build failed because of webpack errors
```

This indicated to me that two npm packages, `pg` and `stock-ticker-symbol` were not installed. I checked `package.json` and discovered that they were not included under dependencies. It seems that when they were added to the project, a developer forgot to run `npm i` with the `--save` flag to include them in `package.json` This should be a pretty simple fix and would probably be best to address sooner rather than later, so devs are more easily able to contribute to the project.

--- a/package.json
+++ b/package.json
@@ -12,7 +12,9 @@
     "eslint": "8.28.0",
     "eslint-config-next": "13.0.5",
     "next": "13.0.5",
+    "pg": "^8.8.0",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
+    "stock-ticker-symbol": "^1.1.0"
   }
 }

